impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -150,13 +150,35 @@ public static void awaitExpectedBrokerCountInClusterViaTopic(Map<String, Object>
 
                         final Boolean isQuorate = promise.getNow(false);
                         if (isQuorate) {
-                            admin.deleteTopics(Set.of(CONSISTENCY_TEST));
+                            deleteTopic(admin);
                         }
                         return isQuorate;
                     });
         }
     }
 
+    /*
+     * There are edge cases where deleting the topic isn't possible. Primarily `delete.topic.enable==false`.
+     * Rather than attempt to detect that in advance (as that requires an RPC) we catch that and return normally
+     * otherwise we rethrow the exception.
+     */
+    private static void deleteTopic(Admin admin) throws InterruptedException, TimeoutException, ExecutionException {
+        try {
+            admin.deleteTopics(Set.of(CONSISTENCY_TEST)).all().get(10, TimeUnit.SECONDS);
+        }
+        catch (ExecutionException ee) {
+            if (ee.getCause() instanceof TopicDeletionDisabledException cause) {
+                log.warn("Failed to delete {}. Caught: {} ", CONSISTENCY_TEST, cause.getMessage(), cause);
```

Comment:
logging at warn as this could cause problems for test authors but equally its their configuration that creates the issue.